src/ClientInterface.php

```diff
@@ -21,151 +28,108 @@ interface ClientInterface
     /**
      * Test connection.
      *
-     * @param string|null $value Any string value (server must returns it back)
+     * @param DevPingParams $params
      *
      * @throws BadRequestException
      * @throws BadResponseException
      *
      * @return DevPingResponse
      */
-    public function devPing(?string $value = null): DevPingResponse;
+    public function devPing(DevPingParams $params): DevPingResponse;
```

Comment:
Specifically in such cases, is it possible to make the parameter `DevPingParams $params` nullable?

```php
 public function devPing(?DevPingParams $params): DevPingResponse;
```

For simple use like in older version.

```php
/** #var ClientInterface $client */
$client->devPing();
```

Reply:
Good idea! I have implemented this f4dbd2e360a8dd364f754e849fb875466a4dcc97

Reply:
great!